frontend/src/lib/k8s/event.ts

```diff
@@ -238,7 +239,19 @@ class Event extends KubeObject<KubeEvent> {
       options?.queryParams ?? {}
     );
 
-    return this.useListForClusters(clusters, { queryParams: queryParameters });
+    const newWarningsList = this.useListForClusters(clusters, { queryParams: queryParameters });
```

Comment:
For later... we should probably investigate doing this change inside of "useListForClusters"? Or I thought this had a caching layer built in?